tiktok.cs

```diff
@@ -5,394 +5,105 @@ namespace First_project
-            {
-                if (board[xGuess, yGuess - 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess, yGuess + 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess - 1, yGuess - 1] == 2)
-            {
-                if (board[xGuess - 2, yGuess - 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess + 1, yGuess + 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess - 1, yGuess] == 2)
-            {
-                if (board[xGuess - 2, yGuess] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess + 1, yGuess] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess - 1, yGuess + 1] == 2)
-            {
-                if (board[xGuess - 2, yGuess + 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess + 1, yGuess - 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess, yGuess + 1] == 2)
-            {
-                if (board[xGuess, yGuess + 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess, yGuess - 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            return false;
-        }
-
-            static void Main(string[] args)
-        {
-
-            
             bool play = true;
-            string playString;
-            while (play == true)
+            while (play)
             {
 
                 int xGuess;
                 int yGuess;
-                int[,] newBoard = new int[,] { { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0 } };
+                int[,] newBoard = new int[,] { {0, 0, 0}, {0, 0, 0}, { 0, 0, 0} };
                 int[,] board = newBoard;
-                int turnCounter = 1;
-                while (turnCounter <= 9)
-                {
-                    
-                    //XTurn
-                    XTurn(out xGuess, out yGuess);
-
-                    if (xGuess > 3)

```

Comment:
These if statements have the same body, so you just have 1 with and or in between the cases